

namespace rhoban
{
    class PlacementController
    {
    private:
        /* data */
    public:
        PlacementController(/* args */);
        ~PlacementController();
    };
    
    PlacementController::PlacementController(/* args */)
    {
    }
    
    PlacementController::~PlacementController()
    {
    }
    
} // namespace rhoban
